Frontend Improvements on Buttons and About Page

sezaiemrekonuk opened the pull request:
I added button animations and styles for Arge and Events pages.

sezaiemrekonuk committed:
new: 404 pagenotfound error

sezaiemrekonuk committed:
refactor: simplified and solved general url

sezaiemrekonuk committed:
new: button animations and styles

furkansimsekli commented:
Do you think the two animations on the cards are a bit unnecessary? I really liked the buttons, maybe we can remove the card getting larger animation. What do you think about that?

ikolomiko commented:
I agree with @furkansimsekli , two animations on the same element would be overkill. The new button animations are better imho. I found the sliding animation a bit too slow, so it could be better if you half the transition durations on `.sliding-element` and `.sliding-element-small`.

ikolomiko commented on views/sections/arge.ejs:
```diff
@@ -1,21 +1,25 @@
                                         <%= project.name %>
                                     </h4>
-                                    <button class="btn events-color mt-auto border-0">Detaylar &gt;</button>
+                                    <button class="btn shadow border-1 button-element-event-special" onclick="location.href='/idea'">
```
The `onclick` attribute here is redundant and should be removed, since the clicks are handled by the parent div. You probably accidentally overlooked that :)

ikolomiko commented on views/sections/events.ejs:
```diff
@@ -15,7 +17,9 @@
                                         <%= event.name %>
                                     </h4>
-                                    <button class="btn events-color mt-auto border-0">Detaylar &gt;</button>
+                                    <button class="btn shadow border-1 button-element-event-special" onclick="location.href='/idea'">
```
Same as [the previous one](https://github.com/hacettepeoyt/website/pull/26#discussion_r1137391656)

ikolomiko reviewed (changes requested):
For my overall review, there are some tiny flaws here and there but they're easily fixable. You've done an amazing job overall. Thank you for your contributions @sezaiemrekonuk , much appreciated! 🎉 Looking forward for the fixes. 👀

sezaiemrekonuk committed:
refactor: requests done

sezaiemrekonuk commented:
I removed on-click events, and reduced animation latencies by half, even though I didn't make it I removed the class that gives card elements scaling (getting larger and more prominent).

furkansimsekli committed:
Merge branch 'frontendchng' of github.com:hacettepeoyt/website into frontendchng

furkansimsekli committed:
frontend.about: add social links for board members

Use justify-content-evenly in the board members' container.
Resolves #20.
